validation/chOpenApiValidator.go

```diff
@@ -98,20 +98,20 @@ func (chv CHValidatorImpl) ValidateRequestAgainstOpenApiSpec(httpReq *http.Reque
 		// Switch off the addition of schema error details to the returned error. This stops the OpenApi schema being added to errors.
 		openapi3.SchemaErrorDetailsDisabled = true
 
-		log.Info("Validating request...", nil)
+		log.InfoC(contextId, "Validating request...", nil)
```

Comment:
would it help to log the schema that you are validating against in the logs .

Reply:
+1to logging the location of the spec we are using. Something like "Validating request using %s", cfg.OpenApiSpec. But wouldn't suggest logging the full spec to the console as it's large